.github/scripts/compare.js

```
@@ -0,0 +1,348 @@
+// kick it off
+let output = `## 📦 Next.js Bundle Analysis for ${PACKAGE_NAME}
+
+This analysis was generated by the [Next.js Bundle Analysis action](https://github.com/hashicorp/nextjs-bundle-analysis). 🤖
+
+`
+
+// pull the global bundle out, we handle this separately
+const globalBundleCurrent = currentBundle.__global
+const globalBundleBase = baseBundle.__global
+delete currentBundle.__global
+delete baseBundle.__global
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)